packages/core/src/components/callout/_callout.scss

```diff
@@ -23,14 +25,16 @@ Styleguide callout
 
 .#{$ns}-callout {
   @include running-typography();
-  position: relative;
+  display: flex;
   border-radius: $pt-border-radius;
   background-color: rgba($gray3, 0.15);
   width: 100%;
   padding: $pt-grid-size ($pt-grid-size * 1.2) ($pt-grid-size * 0.9);
 
   // CSS API support
   &[class*="#{$ns}-icon-"] {
+    display: block;
```

Comment:
look, this line amounts to a special case for our own docs usage of the CSS API, which is a major code smell. the CSS API functions quite differently when you add an icon, which is weird.

all this to say i'm still not sold on the value of this change, and i'm quite uncomfortable with this special case to avoid changing all our docs.

granted, if we used MDX or could somehow generate those callouts using the component then this would not be a problem.